sheetstorm/management/commands/generate_test_data.py

```diff
@@ -186,3 +214,27 @@ def add_arguments(self, parser: Any) -> None:
             type=int,
             help="Indicates the maximum number of completed projects to be in the database",
         )
+        parser.add_argument(
+            "--small",
+            dest=TestDataSize.SMALL.name,
+            action="store_true",
+            help="Use prepared small set to generate test data",
+        )
+        parser.add_argument(
+            "--medium",
+            dest=TestDataSize.MEDIUM.name,
+            action="store_true",
+            help="Use prepared medium set to generate test data",
+        )
+        parser.add_argument(
+            "--large",
+            dest=TestDataSize.LARGE.name,
+            action="store_true",
+            help="Use prepared large set to generate test data",
+        )
+        parser.add_argument(
+            "--extra_large",
+            dest=TestDataSize.EXTRA_LARGE.name,
+            action="store_true",
+            help="Use prepared extra large set to generate test data",
+        )
```

Comment:
What you added here require also adding additional tests to ensure that it works:)

Reply:
These parameters must be validated that they can't be selected together. Please check if there is possibility provided by ArgParser